src/screens/ENSConfirmRegisterSheet.js

```diff
@@ -327,13 +322,6 @@ export default function ENSConfirmRegisterSheet() {
     getReverseRecord();
   }, [accountAddress]);
 
-  useEffect(() => {
```

Comment:
Don't think we needed this as we are already doing this with `ENSAssignRecordsSheet`